#Expyriment 0.9.0 (Python 3.6.5), .xpe-file, coding: cp1252
#date: Sun Apr 14 2019 17:44:05
#sha1: None
#modules: 
#display: size=(1920, 1080), window_mode=False, open_gl=2
#os: uname_result(system='Windows', node='DESKTOP-OVOASGR', release='10', version='10.0.17134', machine='AMD64', processor='Intel64 Family 6 Model 94 Stepping 3, GenuineIntel')
Time,Type,Event,Value,Detail,Detail2
3762,stderr,received,'Traceback (most recent call last):'
3765,stderr,received,'  File "scripts\\mttexp.py", line 224, in <module>'
3765,stderr,received,'    after_is_left_arrows = expyriment.stimuli.Picture("documents/after_is_left_arrows.png")'
3766,stderr,received,'  File "C:\\Users\\annaw\\Anaconda3\\lib\\site-packages\\expyriment\\stimuli\\_picture.py", line 49, in __init__'
3767,stderr,received,'    unicode2byte(filename)))'
3767,stderr,received,"OSError: The picture file 'b'documents/after_is_left_arrows.png'' does not exist"
